src/Dev/SapphireTest.php

```diff
@@ -1090,7 +1090,7 @@ public static function getExtraDataObjects()
      */
     public static function getExtraControllers()
     {
-        return static::$extra_controllers;
+        return array_merge([Security::class], static::$extra_controllers);
```

Comment:
I'm not sure this should be in here. Security has it's own routes that should be routed by default via YAML.

Reply:
This change makes requests to the security controller go directly to the controller rather than via the ratelimiter.

The routes are, by default, defined in yaml (to go via the rate limited middleware). This just replaces that definition.

Reply:
Can you move the rate limit enable / disable to a RateLimitDisableTestState? If you really need to disable it for all tests that would seem the most injectable, instead of baking this into the sapphiretest class.

Reply:
See below.